app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,24 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>EcommerceTrainning</title>
+    <title><%= full_title(yield(:title)) %></title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet"
```

Comment:
không dùng CDN nha em, tham khảo https://ndirangumindo.medium.com/how-to-add-bootstrap-5-to-your-rails-6-application-eab10c879921